.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,23 +10,66 @@ assignees: ''
 
 The release number or commit hash of the version you're using.
 
-## Operating system & version
+## \[Optional\] Setup information
 
-Ex: Windows 10, Ubuntu 18.04, iOS 14.
+If you are reporting a bug about:
+
+- difficulties getting Bevy to build or run on your machine
+- unusual rendering bugs
+- unusual input bugs
+- hardware specific problems
+
+These issues are often specific to hardware or OS configuration and can be challenging to reproduce.
+As a result, please consider including information about:
```

Comment:
Maybe put everything from the start of this section up to this point in an html comment so it doesn't clutter the issue description even if the issue reporter forgets to remove it.

Reply:
Hmm. I think the better tool is write permissions to clean up bad issue descriptions (since I don't want to close valid issues due to bad formatting). This feels like over-optimizing for a single failure mode.